packages/uswds/src/Widgets/UpDownWidget.tsx

```diff
@@ -0,0 +1,11 @@
+import { FormContextType, RJSFSchema, StrictRJSFSchema, WidgetProps } from '@rjsf/utils';
+import TextInputWidget from './TextInputWidget'; // Use TextInputWidget
+
+export default function UpDownWidget<
```

Comment:
The `core` implementation simply uses the `BaseInputTemplate` with the `type='number'` So you can just delete this if you've implemented the `BaseInputTemplate` properly